src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,11 @@
-export const UserInfo = () => {};
+import { FC } from 'react';
+import { User } from '../../types/User';
+
+type Props = {
+  user :User
+};
+export const UserInfo :FC<Props> = ({ user }) => (
+  <a className="UserInfo" href={`mailto:${user.email}`}>
+    {user?.name}
```

Comment:
Not needed, user is present because you do `{todo.user && <UserInfo ...`
```suggestion
    {user.name}
```